helm/managed/logging/elastic/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: elastic4hpcclogs
+description: A Helm chart for launching a lightweight Elastic Stack cluster using Kubernetes
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+
+appVersion: 7.9.3
```

Comment:
As this is our umbrella chart, where the dependencies have their own versions.
I suspect this version should be on we define, e.g. 0.1.0

Reply:
From HELM docs: 
_appVersion: The version of the app that this contains (optional). Needn't be SemVer. Quotes recommended._

This chart resides within our helm dir, but doesn't contain any of our apps, only elastic apps. Also, it's conditional, I might remove it since it isn't really serving any purpose